eox_tenant/management/commands/edit_tenant_values.py

```diff
@@ -0,0 +1,160 @@
+"""
+This module contains the command class to modify the values
+from a tenant so that we can add, remove of edit tenants in bulk.
+"""
+from __future__ import print_function, unicode_literals
```

Comment:
I think print is not being used. @felipemontoya

Reply:
You are right. I was using it everywhere but migrated to LOGGER at the end